tools/make.py

```diff
@@ -264,6 +263,17 @@ def find_depbo_tools():
 
     return requiredToolPaths
 
+def pboproject_settings():
+    """Use registry entries to configure needed pboproject settings."""
+    value_exclude = "thumbs.db,*.txt,*.h,*.dep,*.cpp,*.bak,*.png,*.log,*.pew,source,*.tga"
+
+    try:
+        k = mikero_windows_registry(r"pboProject\Settings", access=winreg.KEY_SET_VALUE)
+        winreg.SetValueEx(k, "m_exclude", 0, winreg.REG_SZ, value_exclude)
+        winreg.SetValueEx(k, "m_exclude2", 0, winreg.REG_SZ, value_exclude)
```

Comment:
`m_exclude2` seems to be the `-B` exclusions, which should be the only way we build, but it's probably safest to set both